app/data_models/questionnaire_store.py

```diff
@@ -10,18 +12,21 @@
 class QuestionnaireStore:
     LATEST_VERSION = 1
 
-    def __init__(self, storage, version=None):
+    def __init__(self, storage: Any, version: int = None):
+        # Storage type hint needs to be Any due to circular import
         self._storage = storage
         if version is None:
             version = self.get_latest_version_number()
         self.version = version
-        self._metadata = {}
+        self._metadata: Any = {}
         # self.metadata is a read-only view over self._metadata
-        self.metadata = MappingProxyType(self._metadata)
-        self.response_metadata = {}
+        self.metadata: Any = MappingProxyType(self._metadata)
```

Comment:
How come these are Any?

Reply:
We use `clear()` method further down in the code on this `metadata` but we declare it here as empty `dict` hence the type hints confusion.

Reply:
Ok, changed it to `Mapping[str, Any] = {}` now, thanks!

Reply:
So yeah, it uses `.clear()` method of a dict so needs to stay as dict I think.